docs/data/material/pagesVarnish.js

```diff
@@ -25,6 +25,8 @@ module.exports = [
   {
     pathname: '/material-ui/varnish/extensions',
     subheader: 'EXTENSIONS',
-    children: [],
+    children: [
+      { pathname: '/material-ui/varnish/varnish-extensions', title: 'Varnish Extensions' },
```

Comment:
i think there will be more extensions... so please rename this file to `varnish-eslint-config`